app/components/file-viewer/canvas-viewer.jsx

```diff
@@ -143,12 +147,12 @@ class CanvasViewer extends React.Component {
           style={Object.assign({}, this.props.style, this.state.canvasStyle)}
         />
         {
-          this.state.hasScore && this.state.score !== null &&
+          this.state.hasMessage && this.state.message !== null &&
           <span
-            ref={(r) => { this.scoreSpan = r; }}
-            className="canvas-renderer-score"
+            ref={(r) => { this.messageSpan = r; }}
```

Comment:
Is `this.messageSpan` used? I don't see it anywhere else in this PR.

Reply:
ah well spotted - removed